packages/alea-frontend/pages/api/job-portal/create-organization-profile.ts

```diff
@@ -59,12 +60,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     incorporationYear,
     isStartup,
     website,
-    domain,
     about,
     companyType,
     officeAddress,
     officePostalCode,
   } = req.body;
+  const domain = userId.split('@')[1] ?? '';
```

Comment:
This might be used in multiple places. Suggest creating a helper function:

```
export function getDomainFromEmail(email: string): string | undefined {
  if (!email) return undefined;
  const parts = email.split("@");
  return parts.length === 2 ? parts[1] : undefined;
}
```